src/index.js

```diff
@@ -23,6 +24,10 @@ const createTestServer = opts => createCert(opts && opts.certificate)
 		};
 
 		app.set('etag', false);
+		app.use(bodyParser.json({ type: 'application/json' }));
+		app.use(bodyParser.text({ type: 'text/plain' }));
+		app.use(bodyParser.urlencoded({ type: 'application/x-www-form-urlencoded' }));
```

Comment:
This gives a deprecation warning since `extended` is not passed.

Reply:
@wtgtybhertgeghgtwtg oops missed that, thanks for letting me know.

Fixed and published `create-test-server@2.3.1`.